### ISSUE TITLE
Aborting Background Requests when Navigating

### ISSUE THREAD

**Issue by 27136773** [manga-download/hakuneko#6]
Import: https://sourceforge.net/p/hakuneko/tickets/7/
----
When switching between mangas, chapters or pages really quick, a new request is started while the old request is still running.
The old request should be aborted to prevent network usage from increasing.
Download Jobs must be respected and should not be aborted.

**Comment by XenonR** [comment]
I do not think this is still possible. I'm my experience requests are processed from a queue. Or if you are talking about the queue I rather feel it should be an enhancement.

**Comment by 27136773** [comment]
It is about image requests through the fetchAPI, it could be done with [AbortControllers](https://developer.mozilla.org/en-US/docs/Web/API/AbortController) when making the request, but its kinda difficult to hook into all events which requires cancellation...
